### DISCUSSION TITLE
Static files

### DISCUSSION THREAD

**Discussion by oussamach98** [nuxt/nuxt#25044] [Questions]
so I have a file named config.js inside the public folder, the reason for that location is I don't want that file to get rendered after I run "npm run generate", but after I run the command I get two "config.js" files one that's I put in public and a rendered one inside "_nuxt" folder named "config.y1TiTW-m.js", and of course when I make the changes inside "config.js" the pages that have direct use with that file don't take effect and when I do the change on "config.y1TiTW-m.js" the pages take effect.

the bottom line is I want the "config.js" file inside the public folder to not get rendered or get a copy of it rendered and when I make the change inside it the pages will take effect.

**Comment by TheAlexLichter** [comment]
To build a nuxt app, all files are copied into `.output`. Hence, changing the "original" files will never work I'm afraid..
